AuditEventProcessor.cpp

```diff
@@ -41,6 +42,12 @@ extern "C" {
 #include <dlfcn.h>
 }
 
+#include <linux/audit.h>
+#include <syscall.h>
+
+#define PROCESS_CREATE_RECORD_TYPE 4688
```

Comment:
I would prefer it if we used numbers in the 10000 range for our custom events.
Also, I would prefer it if we used "AUOMS_EXECVE" as the record name.

Reply:
Changed to 14688 (happy to change this to something else, i.e. 11309 might make more sense).